lib/sycamore/sycamore/query/execution/sycamore_operator.py

```diff
@@ -538,6 +546,8 @@ def script(self, input_var: Optional[str] = None, output_var: Optional[str] = No
 
 class SycamoreTopK(SycamoreOperator):
     """
+    Note: top_k clustering only operators on document level fields. If you try to cluster on the text contents it
```

Comment:
This comment is confusing. Should we just disallow `text_representation` to be passed as a the field to cluster on here?

Reply:
I can add a validation sure

Reply:
Yeah let's disable text_representation here, we need to split this operator anyway